finance_manager/src/main/resources/db/changelog/changeset/insert-users.sql

```diff
@@ -1,9 +0,0 @@
---liquibase formatted sql
---changeset "gs":3
-
-INSERT INTO users (u_email, u_password, u_is_admin)
-VALUES ('Sergey@exadel.com', 'Sergey!2345', true),
-       ('Katia@exadel.com', 'Katia!2345', false),
-       ('Jura@exadel.com', 'Jura!2345', true),
-       ('Anna@exadel.com', 'Anna!2345', false)
-ON CONFLICT DO NOTHING;

```

Comment:
why the sql file has been deleted?

Reply:
Because we decided not to use all these migrations and created new ones. Now we are creating migrations in parallel with creating entities.isided